inst/shiny/server.R

```diff
@@ -898,12 +902,20 @@ shinyServer(function(input, output, session) {
   })
 
   # Event handler for "Upload" button on import page
-  observeEvent(input$uploadData, withConsoleMsgRedirect({
+  observeEvent(input$uploadData,  withConsoleMsgRedirect({
+    
+    .loadOpen ("Please wait...Data is being imported") # show the notification spinner
```

Comment:
Could all messages take the form of "Please wait while data is being importing. See console log for progress."